devtools/create_mortdat/gametext.h

```diff
@@ -406,6 +406,7 @@ const char *gameDataEn[] =  {
 	"Apart from the letter for you I posted, nothing very important!$",
 	"I was very happy she gave me her bound bible as a present$",
 	"It happened fast and she didn't have time to make any particular will$",
+	// FIXME: Spelling incorrect. suprised should be surprised. Fixing changes game data and thus may cause issues
```

Comment:
This is fine. We may regenerate the data.

Reply:
There is another FIXME a fewl7nes above this...not from me though